src/plugin/playhead/index.js

```diff
@@ -0,0 +1,225 @@
+            this._updatePlayheadPosition();
+        };
+
+        this.unFuns = [];
+        let r;
+
+        this._onBackendCreated = () => {
+            this.wrapper = this.wavesurfer.drawer.wrapper;
+
+            if (this.options.draw) {
+                this._createPlayheadElement();
+                window.addEventListener('resize', this._onResize, true);
+                window.addEventListener('orientationchange', this._onResize, true);
+
+                r = this.wavesurfer.on('zoom', this._onResize);
+                this.unFuns.push(r.un);
+            }
+
+            r = this.wavesurfer.on('pause', () => {
+                if ( this.options.returnOnPause ) {
+                    this.wavesurfer.setCurrentTime(this.playheadTime);
+                }
+            });
+            this.unFuns.push(r);
+
+            r = this.wavesurfer.on('seek', () => {
+                if ( this.options.moveOnSeek ) {
+                    this.playheadTime = this.wavesurfer.getCurrentTime();
+                    this._updatePlayheadPosition();
+                }
+            });
+            this.unFuns.push(r);
+
+            this.playheadTime = this.wavesurfer.getCurrentTime();
+        };
+
+        this._onReady = () => {
+            this.wrapper = this.wavesurfer.drawer.wrapper;
+            this._updatePlayheadPosition();
+        };
+    }
+
+    init() {
+        if (this.wavesurfer.isReady) {
+            this._onBackendCreated();
+            this._onReady();
+        } else {
+            let r;
+
+            r = this.wavesurfer.once('ready', this._onReady);
+            this.unFuns.push(r);
+
+            r = this.wavesurfer.once('backend-created', this._onBackendCreated);
+            this.unFuns.push(r);
+        }
+    }
+
+    destroy() {
+        this.unFuns.forEach(f => f());
+        this.unFuns = [];
+
+        window.removeEventListener('resize', this._onResize, true);
+        window.removeEventListener('orientationchange', this._onResize, true);
+    }
+
+    setPlayheadTime(time) {
+        this.playheadTime = time;
+
+        if (!this.wavesurfer.isPlaying()) {
+            this.wavesurfer.setCurrentTime(time);
+        }
+        this._updatePlayheadPosition();
+    }
+
+    _createPointerSVG() {
+        const svgNS = "http://www.w3.org/2000/svg";
+
+        const el = document.createElementNS(svgNS, "svg");
+        const path = document.createElementNS(svgNS, "path");
+
+        el.setAttribute("viewBox", "0 0 33 30");
+        path.setAttribute("d", "M16.75,0 L31.705211,25.4338659 C32.570163,26.8470269 32.1257518,28.6938024 30.7125908,29.5587544 C30.2411619,29.8473013 29.6991813,30 29.1464566,30 L4.35354336,30 C2.69668911,30 1.35354336,28.6568542 1.35354336,27 C1.35354336,26.4472753 1.50624201,25.9052947 1.794789,25.4338659 L16.75,0 L16.75,0 Z");
+        path.setAttribute("transform", "translate(16.750000, 15.500000) scale(1, -1) translate(-16.750000, -15.500000)");
+        path.setAttribute("id", "polygon");
```

Comment:
I've optimised this icon using svgo, and it encoded the transform into the path itself, so you can remove it. Along with the id attribute.
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 33 30">
  <path d="M16.75 31 31.705 5.566A3 3 0 0 0 29.146 1H4.354a3 3 0 0 0-2.56 4.566L16.75 31z" stroke="#979797" fill="..." /></svg>
```

Reply:
thanks, fixed.